src/routes/+layout.svelte

```diff
@@ -1,5 +1,28 @@
-<script>
+<script lang="ts">
```

Comment:
Important call out that we need to specify `lang="ts"` whenever we want to use typescript in the `<script>` tag, which should basically be all the time.